web/blog/tenzir-node-v4.25/index.md

```diff
@@ -0,0 +1,77 @@
+
+<!-- truncate -->
+
+## New Sinks: Snowflake, OpenSearch, and Elasticsearch
+
+Tenzir Node v4.25 offers several new sinks for sending your data. The [`to_snowflake`](/next/tql2/operators/to_snowflake) sink enables seamless writing of data into
+[Snowflake](https://www.snowflake.com) databases, helping users integrate Tenzir with one of the most popular cloud data platforms. Additionally, the
+new [`to_opensearch`](/next/tql2/operators/to_opensearch) and [`to_elasticsearch`](/next/tql2/operators/to_elasticsearch) sink operators allow direct data output to
+[OpenSearch](https://opensearch.org/) and [Elasticsearch](https://www.elastic.co/elasticsearch), two powerful search and analytics engines.
+
+For more information on using Tenzir in combination with these new sinks, check out our integration pages for [OpenSearch](/next/integrations/opensearch),
+[ElasticSearch](/next/integrations/elasticsearch) and [Snowflake](/next/integrations/snowflake)
+
+## TQL2
+
+We continue to enhance TQL2, moving closer to making it the default language for writing pipelines. This release adds numerous improvements and new features to expand TQL2's capabilities.
+
+To simplify TQL2 adoption, we introduced a TQL2-only mode. When you enable this mode, all pipelines run in TQL2, and the Explorer interface automatically selects TQL2 mode. To enable it, set `TENZIR_TQL2=true` in your environment, configure `tenzir.tql2: true` in your settings, or start the node with `tenzir-node --tql2`.
+
+### From and To
+
+With the [`from`](/next/tql2/operators/from) and [`to`](/next/tql2/operators/to) operators, we have ported one of the last major functionalities from TQL1 to TQL2.
+
+The [`from`](/next/tql2/operators/from) operator lets you onboard data from most sources effortlessly. For example, you can write `from "https://example.com/file.json.gz"` to automatically determine the load operator, compression, and format.
+
+Conversely, the [`to`](/next/tql2/operators/to) operator makes it easy to send data to most destinations. For instance, you can write `to "ftps://example.com/file.json.gz"` to automatically determine the save operator, compression, and format.
+
+### IPs and Timestamps
+
+We improved support for working with the native IP and timestamp types in TQL2.
+
+The `in` operator now checks IP or subnet data for subnet membership. For example, to check whether an IP address belongs to a subnet, use an expression like `1.2.3.4 in 1.2.0.0/16`. Similarly, to verify whether a subnet includes another subnet, write `1.2.0.0/16 in 1.0.0.0/8`. A new function, [`subnet(string)`](/next/tql2/functions/subnet), lets you parse strings as subnets.
+
+To streamline time-related computations, we added the [`from_epoch(x:duration) -> time`](/next/tql2/functions/from_epoch) function (#4854), which converts durations to epoch times. Additionally, you can now convert strings to durations with the [`duration(string)`](/next/tql2/functions/duration) function.
```

Comment:
I think we can strip this section out and maybe add some more fluff to the bigger ones above.